packages/invoices/src/InboundShipment/DetailView/GeneralTab.tsx

```diff
@@ -34,14 +36,34 @@ const Expand: FC<{ rowData: InboundShipmentItem }> = ({ rowData }) => {
 };
 
 export const GeneralTabComponent: FC<GeneralTabProps<InboundShipmentItem>> = ({
-  data,
-  columns,
   onRowClick,
 }) => {
   const { pagination } = usePagination();
-  const activeRows = data.filter(({ isDeleted }) => !isDeleted);
   const t = useTranslation('common');
 
+  const { data, sortBy, onSort } = useInboundLines();
```

Comment:
this hook just returns the lines of the inbound lines which I thought was quite nice. Maybe should pass the `id` rather than `useParams`.

Reply:
maybe should.. but the `useParams` is so clever, shame not to use that approach. and it is neater to just call `useInboundLines()` with no params 😁

Reply:
I'm actually realising that it might also be better to *only* keep the `sort`, `filter` and `pagination` states in the URL and `useParams` for that also.. 🤔